roles/zuul/files/jobs/lore.yaml

```diff
@@ -0,0 +1,14 @@
+---
+- job:
+    name: lore
+    node: 'ubuntu-xenial'
+
+    builders:
+      - zuul-git-prep
+      - shell: |
+        ./tests/markdownlint-cli-test.sh
+        ./tests/textlint-test.sh
+        ./tests/shellcheck-test.sh
```

Comment:
I don't think these commands will work, because nothing installed them ahead-of-time. 

I believe that'll be true for all three of these. The signed-off-by only needs git, which is installed, so it should be fine.